gmso/tests/files/restrained_benzene_ua.top

```diff
@@ -0,0 +1,85 @@
+; File Topology written by GMSO at 2022-11-04 11:13:28.179193
+
+[ defaults ]
+; nbfunc	comb-rule	gen-pairs		fudgeLJ	fudgeQQ
+1			2			no		0.0		0.0
+
+[ atomtypes ]
+; name		at.num	mass		charge		ptype	sigma	epsilon
+CH_sp2      6       13.01900     0.00000	A        0.36950     0.41988
+
+[ moleculetype ]
+; name	nrexcl
+Compound	3
+[ atoms ]
+; nr	type	resnr	residue		atom	cgnr	charge	mass
+1       CH_sp2      1       Compound    _CH     1        0.00000    13.01900
+2       CH_sp2      1       Compound    _CH     1        0.00000    13.01900
+3       CH_sp2      1       Compound    _CH     1        0.00000    13.01900
+4       CH_sp2      1       Compound    _CH     1        0.00000    13.01900
+5       CH_sp2      1       Compound    _CH     1        0.00000    13.01900
+6       CH_sp2      1       Compound    _CH     1        0.00000    13.01900
+
+[ bonds ]
+; ai	aj		funct	b0		kb
+2       1       1           0.14000        0.00000
+6       1       1           0.14000        0.00000
+3       2       1           0.14000        0.00000
+4       3       1           0.14000        0.00000
+5       4       1           0.14000        0.00000
+6       5       1           0.14000        0.00000
+
+[ bonds ]
```

Comment:
based on the other restraints (angle/dihedral) should this not be `bond _restraints`, as the bonds section is listed above, 2 times total?

Reply:
yeah, unlike the angle and dihedral restraints, harmonic potential restraint is under the same directive of bond (so just bonds) but with a different equation, I had them under two sections (same directives just for the ease of reading/writing, but seems like something else is going wrong.